modules/olm-installing-opm.adoc

```diff
@@ -22,91 +15,32 @@ You can install the `opm` CLI tool on your Linux, macOS, or Windows workstation.
 
 .Procedure
 
-ifndef::openshift-origin[]
-. Set the `REG_CREDS` environment variable to the file path of your registry credentials for use in later steps. For example, for the `podman` CLI on Linux:
-+
-[source,terminal]
-----
-$ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
-----
+. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/[OpenShift mirror site] and download the latest version of the tarball that matches your operating system.
```

Comment:
When I cherrypick this to `enterprise-4.6`, will update `latest/` here to be `latest-4.6/` instead.

Reply:
Maybe too picky here. I don't think the Windows Zip file is called a _tarball_. 
s/tarball/archive ?

Reply:
They've still got `.tar.` in the file name for these Windows archives, FWIW: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/

Reply:
I could just call them tar achives, or just archives, tho.

Reply:
NM! I was looking at the wrong download. Carry on!